### ISSUE TITLE
May be a small bug...

### ISSUE THREAD

**Issue by Jiaozrr** [tsing90/pytorch_semantic_human_matting#25]
I tried several times to train the M_net, but it always failed to generate corect alpha. I find a possible mistake in the 36th line of  _pytorch_semantic_human_matting/model/network.py_. It is written that **bg, fg, unsure = torch.split(trimap, 1, dim=1)**. However, after generating some images to show the bg, fg and unsure, I found fg and unsure seemed to be reversed. I changed this line of code into: **bg, unsure, fg = torch.split(trimap, 1, dim=1)** and fixed my problem.

**Comment by CabbageWust** [comment]
Hi ， can you share the datasets with me ? @

**Comment by CabbageWust** [comment]
@Jiaozrr   Hi ， can you share the datasets with me ?

**Comment by Jiaozrr** [comment]
> @Jiaozrr Hi ， can you share the datasets with me ?
Hi,
I use the dataset publicly posted at https://github.com/aisegmentcn/matting_human_datasets, mentioned in another implementation by lizhengwei1992.

**Comment by CabbageWust** [comment]
@Jiaozrr  Thx

**Comment by tobechao** [comment]
@Jiaozrr Hi, What is the effect of your training t-net， I use the aifenge data training effect is very poor？

**Comment by Jiaozrr** [comment]
> @Jiaozrr Hi, What is the effect of your training t-net， I use the aifenge data training effect is very poor？

Hi,
I also get a poor t-net. I guess this dataset is not good enough.

**Comment by sanshibayuan** [comment]
I agree with you, I think that's why I got very low loss at the begining when training M-Net 
Thanks for pointing this out!

**Comment by sanshibayuan** [comment]
I think you're right. But I found the author's comment in another SHM repo 
https://github.com/lizhengwei1992/Semantic_Human_Matting/issues/10

I'm not sure what he means……
